src/writer/autogen.py

```
@@ -253,19 +335,200 @@
+                "role": "user",
+                "content": "A tool call is required.",
+            }
+        )
+    try:
+        calls_by_function = _split_tool_calls(tool_calls)
+    except ValueError as exception:
+        return autogen_state["messages"].append({
+            "role": "user",
+            "content": "An error occurred. " + repr(exception),
+        })
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)